internetbs.php

```diff
@@ -720,8 +720,13 @@ public function addService(
             // Get request fields
             $params = $this->getFieldsFromInput((array) $vars, $package);
 
-            // Register domain
-            $this->registerDomain($vars['Domain'], $row->id ?? null, $params);
+            // Register or transfer domain
+            if (isset($vars['transfer']) || isset($vars['transferAuthInfo'])) {
+                $vars['epp_code'] = $vars['transferAuthInfo'] ?? null;
+                $this->transferDomain($vars['Domain'], $row->id ?? null, $params);
```

Comment:
Okay, this is better, but the transferDomain() method doesn't actually work.  It fails to submit the contact data required by the api.